examples/c/reflection/dser_type/src/main.c

```diff
@@ -0,0 +1,51 @@
+#include <dser_type.h>
+
+#include <stdio.h>
+
+
+typedef struct {
+    ecs_vec_t v_i32; // vector<ecs_i32_t>
+    ecs_vec_t vv_i32; // vector<vector<ecs_i32_t>>
+} MiscellaneousVectors;
+
+
+
+int main(int argc, char *argv[]) {
+    ecs_world_t *ecs = ecs_init_w_args(argc, argv);
+
+    ECS_COMPONENT(ecs, MiscellaneousVectors);
+
+    ecs_entity_t VectorI32 = ecs_vector(ecs, {.type = ecs_id(ecs_i32_t)});
+    ecs_entity_t VectorVectorI32 = ecs_vector(ecs, {.type = VectorI32});
+    ecs_entity_t t = ecs_struct_init(ecs, &(ecs_struct_desc_t){
```

Comment:
Inconsistency between naming of vector types (PascalCase) and the struct type. Suggestion to rename the vector types to `v_i32` and `vv_i32`.

Reply:
Fixed